lib/output/mongodb.go

```diff
@@ -8,9 +8,9 @@ import (
 
 // MongoDBConfig contains config fields for the MongoDB output type.
 type MongoDBConfig struct {
-	MongoConfig client.Config `json:",inline" yaml:",inline"`
+	ClientConfig client.Config `json:",inline" yaml:",inline"`
 
-	Operation    string              `json:"operation" yaml:"operation"`
+	Operation    client.Operation    `json:"operation" yaml:"operation"`
```

Comment:
I know it's super picky and extremely unlikely to be worth it, but unfortunately we can't change these field names/types without a major version release as they're public. It's lame, but in V4 it'll all be private and then we can refactor to our heart's content.

Reply:
But just to clarify, I think it's good to have an explicit `client.Operation` type but we'll need to convert to/from string after config parse.

Reply:
Oops, indeed! I need to contribute more often, so I don't forget the gotchas 😁  Thanks for spotting that!